docs/source/contributor-guide/quarterly_roadmap.md

```diff
@@ -21,6 +21,12 @@
 
 A quarterly roadmap will be published to give the DataFusion community visibility into the priorities of the projects contributors. This roadmap is not binding.
 
+## 2023 Q4
+
+- Improve data output (`COPY`, `INSERT` and DataFrame) output capability [#7079](https://github.com/apache/arrow-datafusion/issues/7079)
```

Comment:
It might be better to reference the relevant epic issue here instead #6569

Reply:
```suggestion
- Improve data output (`COPY`, `INSERT` and DataFrame) output capability [#6569](https://github.com/apache/arrow-datafusion/issues/6569)
```